trunk/conf/full.conf

```diff
@@ -2317,7 +2318,7 @@ vhost example.transcode.srs.com {
             #       [999], replace this const to current millisecond.
             #       [timestamp],replace this const to current UNIX timestamp in ms.
             # @remark we use golang time format "2006-01-02 15:04:05.999" as "[2006]-[01]-[02]_[15].[04].[05]_[999]"
-            output rtmp://127.0.0.1:[port]/[app]/[stream]_[engine]?vhost=[vhost];
+            output rtmp://127.0.0.1:[port]/[app]/[stream]_[engine]?vhost=[vhost][param];
```

Comment:
What format is param? Maybe your rtmp url is incorrect. For example, 
`param="?vhost=site.streamer&secret=09mY6AsPSeSF"`

Reply:
resolved